rtb/verbali/esterni/VE_2024-11-27_ver1.0.0.typ

```diff
@@ -0,0 +1,77 @@
+
+= Sintesi dell'incontro 
+L'incontro si è svolto tra la conclusione del primo #rifGlossario("sprint") e l'inizio del secondo, ed è stato di carattere prevalentemente discorsivo. Durante il _meeting_ si è discusso principalmente dei vari punti:
+
+- È stata aggiornata l'azienda sui progressi compiuti nello sprint precedente, che ha avuto come focus la redazione della documentazione.
+- Sono stati definiti congiuntamente gli obiettivi per il prossimo periodo.
+
+Andrea Precoma aveva pronta una prima bozza del simulatore dati. L'azienda, tuttavia, ha preferito rimandare al prossimo incontro, consigliando di implementare una tecnica di "containerizzazione" con #rifGlossario("Docker"). Questo principalmente per permetterci di iniziare a sperimentare con questa tecnologia, nuova a molti membri del gruppo, ma anche per facilitare la collaborazione sul codice.
+
+Successivamente l'azienda ha risposto alle domande del gruppo.
+
+
+= Risposte alle domande  
+Durante la stesura del documento Analisi dei requisiti, sono sorti vari dubbi a cui il gruppo non era sicuro di saper dare una risposta corretta. Sono riportati, in dettaglio, qui di seguito.
+== Sensori come attori
+Il gruppo non comprendeva se i sensori dovessero essere interpretati come attori.
+
+Il partecipante esterno di Sync Lab, Andrea Dorigo, ha chiarito che, poiché il prodotto comprende esclusivamente il _software_, i sensori devono essere considerati attori esterni. Se il prodotto avesse incluso sia il sensore che il _software_ integrato, la situazione sarebbe stata diversa, ma non è questo il caso.
+
+== Registrazione utenti
+Un altro dubbio sorto durante l'Analisi dei requisiti era: è necessario creare una logica di login per gli utenti e per l'amministratore ?
+
+Dialogando insieme abbiamo capito che:
+- Non è necessario prevedere un sistema di login per gli utenti, dato che il prodotto attuale prevede esclusivamente l'accesso alla dashboard da parte dell'amministratore.
+
+- Per l'amministratore, l'autenticazione sarà opzionale e supportata tramite #rifGlossario("Grafana"), che integra già un sistema di autenticazione.
+
+
+Ci è stato quindi riconfermato che il focus del prodotto sarà sull'utilizzo lato amministratore, e non lato utente, di cui ci interesseranno solamente i dati personali e di spostamento.
+
+== Nifi
+Sono stati richiesti, da parte del gruppo, alcuni consigli riguardo ad #rifGlossario("Apache Nifi"), in quanto nessun membro del gruppo ne è particolarmente esperto.
+
+L'azienda si è dimostrata ancora una volta molto disponibile, condividendo subito del materiale web e proponendo di fissare una sessione di #rifGlossario("Deep Dive") a riguardo, più avanti.
+
+= Conclusioni  
+L'incontro è stato utile: confrontarsi sui vari dubbi permette sempre di comprendere al meglio la direzione ottimale per il progetto.
+
+Al prossimo incontro, previsto per mercoledì 4 dicembre, sarà presentata la prima versione del simulatore dati.
+
+
+= Tabella delle decisioni
```

Comment:
Probabilemente la decisione di spendere un po' di tempo a fare concretamente la repo del codice e la containerizzazione con docker è una decisione che potrebbe essere messa in questo verbale invece che su quello interno, confrontarsi con gli altri

Reply:
Ho chiesto nel gruppo. In caso dicano di metterlo qua lo cambio, intanto pusho così ti guardi le altre correzioni

Reply:
Il gruppo ha parlato

Reply:
Invece per la repo del codice non penso interessi all'azienda

Reply:
Di là si può lasciare come da conversazione telegram.